packages/flutter/lib/src/material/tabs.dart

```diff
@@ -607,11 +608,14 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
     this.unselectedLabelColor,
     this.unselectedLabelStyle,
     this.dragStartBehavior = DragStartBehavior.start,
+    this.overlayColor,
     this.mouseCursor,
+    this.enableFeedback = true,
```

Comment:
If this was a `bool?` and defaulted to null, then in the future the value could come from a theme.